### ISSUE TITLE
LLVM installation fail for x86_64-unknown-freebsd

### ISSUE THREAD

**Issue by Asthowen** [taiki-e/setup-cross-toolchain-action#15]
Hello!

When I try to build for `x86_64-unknown-freebsd`, LLVM installation fails:
```
Run taiki-e/setup-cross-toolchain-action@v1
  with:
    target: x86_64-unknown-freebsd
Run bash --noprofile --norc "${GITHUB_ACTION_PATH:?}/main.sh"
  bash --noprofile --norc "${GITHUB_ACTION_PATH:?}/main.sh"
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    INPUT_TARGET: x86_64-unknown-freebsd
    INPUT_RUNNER: 
Install toolchain
  Unable to find image 'ghcr.io/taiki-e/rust-cross-toolchain:x86_64-unknown-freebsd-dev-amd64' locally
  x86_64-unknown-freebsd-dev-amd64: Pulling from taiki-e/rust-cross-toolchain
  01085d60b3a6: Already exists
  0c9bca99f601: Pulling fs layer
  0c9bca99f601: Verifying Checksum
  0c9bca99f601: Download complete
  0c9bca99f601: Pull complete
  Digest: sha256:925f9e88bb62ec0b5f1b11034f4c1ee611785611b263f7500545248aba2d00e4
  Status: Downloaded newer image for ghcr.io/taiki-e/rust-cross-toolchain:x86_64-unknown-freebsd-dev-amd64
  b6447ebdefa5f1ca89a998803db27b91eea1de7ead4beec9ecc81606e54f9a2b
Install LLVM
  E: Error, pkgProblemResolver::Resolve generated breaks, this may be caused by held packages.
  E: Error, pkgProblemResolver::Resolve generated breaks, this may be caused by held packages.
  E: Error, pkgProblemResolver::Resolve generated breaks, this may be caused by held packages.
  E: Error, pkgProblemResolver::Resolve generated breaks, this may be caused by held packages.
  E: Error, pkgProblemResolver::Resolve generated breaks, this may be caused by held packages.
  E: Error, pkgProblemResolver::Resolve generated breaks, this may be caused by held packages.
  E: Error, pkgProblemResolver::Resolve generated breaks, this may be caused by held packages.
  E: Error, pkgProblemResolver::Resolve generated breaks, this may be caused by held packages.
  E: Error, pkgProblemResolver::Resolve generated breaks, this may be caused by held packages.
  E: Error, pkgProblemResolver::Resolve generated breaks, this may be caused by held packages.
  E: Error, pkgProblemResolver::Resolve generated breaks, this may be caused by held packages.
  Error: Process completed with exit code 100.
  ```

**Comment by taiki-e** [comment]
It appears to be a (probably temporary) bug in the GitHub action runner or apt.llvm.org, and can be worked around by using ubuntu-20.04 runner instead of ubuntu-22.04/ubuntu-latest runner.

- ubuntu-22.04/ubuntu-latest (fail) https://github.com/taiki-e/setup-cross-toolchain-action/actions/runs/5686204233/job/15412589330
- ubuntu-20.04 (pass) https://github.com/taiki-e/setup-cross-toolchain-action/actions/runs/5686204233/job/15412589402

**Comment by taiki-e** [comment]
Probably related: https://github.com/llvm/llvm-project/issues/64120

**Comment by Asthowen** [comment]
I've just tried again, it works if I run `sudo apt update && sudo apt upgrade -y` before installing llvm.

**Comment by taiki-e** [comment]
Thanks! I published 1.12.1, including that workaround.
